packages/front/src/pages/Index/children/Home/EditTaskForm.tsx

```diff
@@ -7,6 +7,12 @@ import {
   useMutation,
   usePreloadedQuery,
 } from "react-relay";
+import {
+  inputClassName,
+  selectClassName,
+  StyledButton,
+  StyledForm,
+} from "../../../../shared/ui";
```

Comment:
i think it's better to decide what your goal with that shared/ui package.

has components or util classes?

imho components are better, prevent developers to do more mistakes! Check https://beta.reactjs.org/reference/react/forwardRef to create better ui reusable components!